apps/modernization-ui/src/apps/search/advancedSearch/AdvancedSearch.tsx

```diff
@@ -497,22 +497,10 @@ export const AdvancedSearch = () => {
                                 <div
                                     tabIndex={0}
                                     id="resultsCount"
-                                    aria-label={
-                                        ((lastSearchType === SEARCH_TYPE.PERSON && patientData?.total?.toString()) ||
-                                            (lastSearchType === SEARCH_TYPE.INVESTIGATION &&
-                                                investigationData?.total?.toString()) ||
-                                            (lastSearchType === SEARCH_TYPE.LAB_REPORT &&
-                                                labReportData?.total?.toString())) +
-                                        ' amount of results have been found'
-                                    }
+                                    aria-label={resultTotal + ' amount of results have been found'}
```

Comment:
suggestion: The `aria-label` message is a little confusing and should match what's displayed more.  `X Results for..` is displayed, should the message be similar? `X Results have been found`